readme.md

```diff
@@ -8,16 +8,87 @@ This is a telegram bot which can:
 Future feature plans:
 - FA notifications
 
-## Setup
+You can use it freely at [@FASearchBot](https://t.me/FASearchBot), add it to your own chats and channels, or use it inline anywhere by mentioning it
+
+## Commands
+
+### Neaten
+
+Private message it a link to a Fur Affinity submission page, thumbnail url or image download url, i.e, a link beginning with `http://?.facdn.net/art/` and it will reply with the image's thumbnail and a link to that image's gallery page.
+
+### Inline Responses
+
+In any chat, mention the bot and pass it an FA query
+`@FASearchBot [query]` and pick from the inline results which image you'd like to share.
+
+You can also use these qualifiers inline:
+`@FASearchBot gallery:username`
+`@FASearchBot scraps:username`
+`@FASearchBot favs:username`
+
+### Subscriptions
+
+`/add_subscription [query]`
+`/list_subscriptions`
+`/remove subscription [query]`
+
+`/add_blocklisted_tag [tag]`
+`/list_blocklisted_tag [tag]`
+`/remove_blocklisted_tag [tag]`
+
+### Search Queries
+
+The bot will support all Fur Affinity queries, and some extra filter options:
+
+Taken from FurAffinity's search documentation: 
```

Comment:
Ah, these search options actually only work in the inline query at the moment, and don't work in subscriptions yet. Only the negation and rating work in subscriptions unfortunately